com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/parser/DockerInfoParser.java

```diff
@@ -0,0 +1,48 @@
+package com.sap.cloud.lm.sl.cf.core.parser;
+
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.client.lib.domain.DockerCredentials;
+import org.cloudfoundry.client.lib.domain.DockerInfo;
+
+import com.sap.cloud.lm.sl.mta.util.PropertiesUtil;
+
+public class DockerInfoParser implements ParametersParser<DockerInfo> {
+
+    private static final String DOCKER = "docker";
+
+    @Override
+    public DockerInfo parse(List<Map<String, Object>> parametersList) {
+        Map<String, String> dockerParams = getDockerParams(parametersList);
+        DockerInfo dockerInfo = getDockerInfo(dockerParams);
```

Comment:
Return the value directly instead of assigning it to a variable. We have a tool that checks our code for [code smells](https://en.wikipedia.org/wiki/Code_smell) and this would be caught as such. See:
https://stackoverflow.com/questions/31733811/local-variables-before-return-statements-does-it-matter